app/components/SearchBox.jsx

```diff
@@ -16,33 +16,26 @@ export default class SearchBox extends React.Component {
   }
 
   handleSearchFilterChange() {
-    this.props.handleUserInput(
-      this._searchFilter.value
-    );
+    this.props.handleUserInput(this._searchFilter.value);
   }
 
   render() {
-    let placeholder = 'Search ' + this.props.templateCount + ' community contributed templates...';
+    let placeholder = "Search " + this.props.templateCount + " community contributed templates...";
     return (
       <section className="template-search">
         <div className="container">
           <div className="row clearfix">
             <div className="column two-thirds">
               <div className="search-box">
                 <div className="search-input">
-                  <input autoFocus
-                      onChange={this.handleSearchFilterChange}
-                      placeholder={placeholder}
-                      ref={(c) => this._searchFilter = c}
-                      type="text"
-                      value={this.props.filterText}
-                  />
+                  <input autoFocus onChange={this.handleSearchFilterChange} placeholder={placeholder} ref={(c) => (this._searchFilter = c)} type="text" value={this.props.filterText} />
```

Comment:
At a certain number of props, probably 2 in my mind I prefer each prop having it's own line. Happy with this as is but it can get a bit hard to read